include/deal.II/grid/tria_accessor.h

```diff
@@ -1022,17 +1022,20 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
    * Return a constant reference to the manifold object used for this object.
    * This function exists for backward compatibility and calls get_manifold()
    * internally.
+   *
+   * @deprecated The classes derived from Boundary have been deprecated in
+   * favor of the Manifold classes.
    */
-  const Boundary<dim,spacedim> &get_boundary () const;
+  DEAL_II_DEPRECATED const Boundary<dim,spacedim> &get_boundary () const;
 
   /**
    * Return a constant reference to the manifold object used for this object.
    *
-   * As explained in
-   * @ref boundary "Boundary and manifold description for triangulations",
-   * the process involved in finding the appropriate manifold description
-   * involves querying both the manifold or boundary indicators. See there for
-   * more information.
+   * As explained in the
+   * @ref manifold
+   * module, the process involved in finding the appropriate manifold
+   * description involves querying both the manifold or boundary
+   * indicators. See there for more information.
```

Comment:
This isn't going to be true for long anymore. Can you make a note somewhere that we need to update this location?

Reply:
Yup; see #6200 :)